service/gcsutils/tarlib/tardisk.go

```diff
@@ -126,7 +126,7 @@ func CreateTarDisk(in io.Reader,
 
 	logrus.Info("entering CreateTarDisk")
 
-	mntFolder, err := ioutil.TempDir(tmpdir, "mnt")
```

Comment:
Same here. The real path is `tmpdir`  not `mnt` or `tmp`. Is this causing problems for the read only root file system? `tmpdir` is in a path backed by the scratch disk.